src/adapters/postgres/fields-adapter.ts

```diff
@@ -281,25 +284,159 @@ export class PostgresFieldsService implements IServicelocatorfields {
     }
 
 
-    public async search(dtoFileName){
+    public async search(dtoFileName) {
         let { limit, page, filters } = dtoFileName;
-    
+
         let offset = 0;
         if (page > 1) {
             offset = parseInt(limit) * (page - 1);
         }
-        
+
         if (limit.trim() === '') {
             limit = '0';
         }
-    
+
         const whereClause = {};
         if (filters && Object.keys(filters).length > 0) {
             Object.entries(filters).forEach(([key, value]) => {
                 whereClause[key] = value;
             });
         }
-        return {offset,limit,whereClause};
-      }
+        return { offset, limit, whereClause };
+    }
+
+    public async getFieldOptions(request: any, fieldName: string, controllingfieldfk: string, response: Response) {
+        const apiId = APIID.FIELDVALUES_SEARCH;
+        let context = 'COHORT';
```

Comment:
@souravbhowmik1999 @Shubham4026 Why context and context type are harcoded in field adpter?

Reply:
@souravbhowmik1999 This should be Harcoded Wherever we are calling the function. Please Check.

@vaivk369 Hardcoding "COHORT" is necessary But we will hardcode it in Cohort Member API.

Reply:
YES
@Shubham4026 @souravbhowmik1999